src/Symfony/Component/Workflow/Workflow.php

```diff
@@ -30,10 +30,12 @@
  * @author Fabien Potencier <fabien@symfony.com>
  * @author Grégoire Pineau <lyrixx@lyrixx.info>
  * @author Tobias Nyholm <tobias.nyholm@gmail.com>
+ * @author Carlos Pereira De Amorim <carlos@shauri.fr>
  */
 class Workflow implements WorkflowInterface
 {
     public const DISABLE_ANNOUNCE_EVENT = 'workflow_disable_announce_event';
+    public const DEFAULT_INITIAL_CONTEXT = ['initial' => true];
```

Comment:
Is there a reason you made this an associative array and not an indexed one? 
`public const DEFAULT_INITIAL_CONTEXT = ['initial'];`

I see in one of your tests you did use an indexed array as $context.

Are yuo

Reply:
this way, you can do :
` if ($context['initial']??false) `